RevenueCat/Scripts/Purchases.cs

```diff
@@ -41,9 +42,21 @@ public partial class Purchases : MonoBehaviour
     /// </summary>
     public delegate void GetPaymentDiscountFunc(PaymentDiscount paymentDiscount, Error error);
 
-    [Tooltip("Your RevenueCat API Key. Get from https://app.revenuecat.com/")]
+    [FormerlySerializedAs("revenueCatAPIKey")]
+    [ObsoleteAttribute("This property is obsolete. Use revenueCatAPIKeyIOS and revenueCatAPIKeyAndroid instead.", false)]
+    [Tooltip("(DEPRECATED) RevenueCat API Key. Get from https://app.revenuecat.com/. " +
+             "This property is obsolete. " +
+             "Use revenueCatAPIKeyIOS and revenueCatAPIKeyAndroid instead.")]
     // ReSharper disable once InconsistentNaming
-    public string revenueCatAPIKey;
+    public string deprecatedLegacyRevenueCatAPIKey;
+
+    [Tooltip("RevenueCat API Key specifically for iOS. Get from https://app.revenuecat.com/")]
+    // ReSharper disable once InconsistentNaming
+    public string revenueCatAPIKeyIOS;
```

Comment:
maybe we should name these `revenueCatAPIKeyApple`/`Google`, since the apple one doesn't have to be ios?

Reply:
good idea. I'd started with `iOS` since we don't support macOS for Unity, but consistency with the dashboard is a great argument

Reply:
💯 on `Google` because we support other Android stores

Reply:
updated!